homeassistant/components/asuswrt/router.py

```diff
@@ -354,6 +372,15 @@ async def _update_unpolled_sensors(self) -> None:
             if self._sensors_data_handler.update_device_count(self._connected_devices):
                 await coordinator.async_refresh()
 
+    async def _get_available_temperature_sensors(self):
+        """Check which temperature information is available on the router."""
+        try:
+            availability = await self._api.async_find_temperature_commands()
+        except (OSError, ValueError):
+            return []
```

Comment:
Are you sure that this are the only exceptions that can be raised? Because this run during initialization, if there are some others error all integration will be unavailable.
May be make sense add also generic exception to avoid this kind of issue?

Reply:
The library is handling exceptions. Line 379 could be cautious and catch any `Exception`. It might be considered too broad in this repo?

Reply:
`************* Module homeassistant.components.asuswrt.router
homeassistant/components/asuswrt/router.py:379:15: W0703: Catching too general exception Exception (broad-except)`

pylint is warning about it too

Reply:
You should add `# pylint: disable=broad-except`.
But I just want to be sure that check for a sensor will not cause whole integration to fail. If you are sure that library is properly managing any condition, you can avoid this.